packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5352,6 +5347,51 @@ Client authentication credentials for an [OAuth 2.0 Authorization Server (AS)](h
 }
 ```
 
+```json
+{
+  "protocol": {
+    "type": "OIDC",
+    "credentials": {
+      "client": {
+        "client_id": "your-client-id",
+        "token_endpoint_auth_method": "private_key_jwt"
+      }
+    }
+  }
+}
+```
+
+###### OpenID Connect Signing Credentials object
+
+Determines the [IdP Key Credential](#identity-provider-key-credential-object) used to sign requests sent to the IdP. Only needed if `token_endpoint_auth_method` is `private_key_jwt`.
+
+| Property | Description                                                                                                    | DataType | Nullable | Readonly  | Validation                                 |
+| -------- | -------------------------------------------------------------------------------------------------------------  | -------- | -------- | --------  | ------------------------------------------ |
+| kid      | [IdP Key Credential](#identity-provider-key-credential-object) reference to Okta's X.509 signature certificate. | String   | TRUE    | FALSE     | Valid Signing Key ID reference             |
+| alg      |The algorithm used when generating the JWT from the private key for token endpoint authentication.  | `RS256`, `RS384`, `RS512`   | FALSE    | FALSE     | Valid date type             |
+
+> **Note:** `kid` is required for a UPDATE request. For an CREATE request, it can be null.
+
+
+```json
+{
+  "protocol": {
+    "type": "OIDC",
+    "credentials": {
+      "client": {
+        "client_id": "your-client-id",
```

Comment:
```suggestion
        "client_id": "{{clientId}}",
```

Reply:
I'm using `your-client-id` because that is what we currently have for API docs, https://developer.okta.com/docs/reference/api/idps/#saml-2-0-trust-credentials-object
Should I leave them as is for consistency?

Reply:
no that really should have been caught in a review. we avoid using references like that and prefer to use variables.